ver_03/script.js

```diff
@@ -0,0 +1,306 @@
+  return playerOneChoice;
+};
+
+var playerOneDiceDecisionK = function () {
+  var diceRoll_1 = playerOneRoll[0];
+  var diceRoll_2 = playerOneRoll[1];
+
+  var playerOneDecisionK =
+    "Player One, you have chosen to keep the dice." +
+    "<br><br> New dice one: " +
+    diceRoll_1 +
+    " <br>& " +
+    " <br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>Please wait for Player Two to complete their turn before <br>typing <u><b>'result'</b></u> to get the comparison.";
+
+  return playerOneDecisionK;
+};
+var playerOneDiceDecisionS = function () {
+  var diceTemp = playerOneRoll[0];
+  playerOneRoll[0] = playerOneRoll[1];
+  playerOneRoll[1] = diceTemp;
+
+  var diceRoll_1 = playerOneRoll[0];
+  var diceRoll_2 = playerOneRoll[1];
+
+  var playerOneDecisionS =
+    "Player One, you have chosen to swap the dice." +
+    "<br><br>New dice one: " +
+    diceRoll_1 +
+    " <br>& " +
+    " <br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>Please wait for Player Two to complete their turn before <br>typing <u><b>'result'</b></u> to get the comparison.";
+
+  return playerOneDecisionS;
+};
+
+var playerTwoDiceRoll = function () {
+  var diceRoll_1 = randomDiceRoll();
+  var diceRoll_2 = randomDiceRoll();
+
+  var playerTwoChoice =
+    "Player Two, you have rolled the following: " +
+    "<br><br>Dice one: " +
+    diceRoll_1 +
+    " <br>& " +
+    " <br>Dice two: " +
+    diceRoll_2 +
+    "<br> Would you like to keep or swap their positions? <br><br><br>Type <u><b>'keep'</b></u> to use the current configuration.<br><br>Type <u><b>'swap'</b></u> to change their positions";
+  playerTwoRoll.push(diceRoll_1, diceRoll_2);
+  console.log(playerTwoRoll);
+  return playerTwoChoice;
+};
+
+var playerTwoDiceDecisionK = function () {
+  var diceRoll_1 = playerTwoRoll[0];
+  var diceRoll_2 = playerTwoRoll[1];
+
+  var playerTwoDecision =
+    "Player Two, you have chosen to swap the dice." +
+    "<br><br>New dice one: " +
+    diceRoll_1 +
+    " <br>& " +
+    " <br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>If Player One has completed their turn, <br>type <u><b>'result'</b></u> to get the comparison";
+
+  return playerTwoDecision;
+};
+
+var playerTwoDiceDecisionS = function () {
+  var diceTemp = playerTwoRoll[0];
+  playerTwoRoll[0] = playerTwoRoll[1];
+  playerTwoRoll[1] = diceTemp;
+
+  var diceRoll_1 = playerTwoRoll[0];
+  var diceRoll_2 = playerTwoRoll[1];
+
+  var playerTwoDecision =
+    "Player Two, you have chosen to swap the dice." +
+    "<br><br>New dice one: " +
+    diceRoll_1 +
+    "<br>& " +
+    "<br>New dice two: " +
+    diceRoll_2 +
+    " <br><br>If Player One has completed their turn, <br>type <u><b>'result'</b></u> to get the comparison.";
+
+  return playerTwoDecision;
+};
+
+var compareResults = function () {
+  var playerOneDice = JSON.stringify(playerOneRoll);
+  var playerTwoDice = JSON.stringify(playerTwoRoll);
+  playerOneDice = Number.playerOneDice;
+  playerTwoDice = Number.playerTwoDice;
```

Comment:
As we discussed, we shouldn't be using `JSON.stringify`, mayb `.join`. and also `Number()` instead of `Number.`